tests/integration/assessment/test_jobs.py

```diff
@@ -22,3 +22,13 @@ def test_job_crawler(ws, make_job, inventory_schema, sql_backend):
 
     assert len(results) >= 1
     assert int(results[0].job_id) == new_job.job_id
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_job_run_crawler(ws, make_job, inventory_schema, sql_backend):
+    # new_job = make_job(spark_conf=_SPARK_CONF)
+    job_run_crawler = SubmitRunsCrawler(ws=ws, sbe=sql_backend, schema=inventory_schema, num_days_history=30)
+    job_runs = job_run_crawler.snapshot()
+
+    assert len(job_runs) >= 1
```

Comment:
don't assert just for job runs number - assert for the job run you expect to have run (e.g. with matching config)